### PR TITLE
Add visual relationships component to unified experience

### PR DESCRIPTION
**Pull request by jasonyavorsky**
From Slack discussions at:
- https://sourcegraph.slack.com/archives/C0LJP8WTB/p1635455873039900
- https://sourcegraph.slack.com/archives/C0LJP8WTB/p1636885061117800

Visualization features like this can be a way to build a really interesting unified capability within Code Graph.

The section is getting a little long, but is fine for now.

### CODE REVIEW

**Comment by jasonyavorsky** [comment]
Cc @lguychard @benvenker @macraig

File: content/company/strategy/code-graph/index.md [hunk 1]
```diff
@@ -68,7 +68,11 @@ How our product works in terms of pricing, upgrade paths, auditing, single sign-
...
+Our product solves real problems today, but investing in the polish, user experience, and cross-feature functionality within our product teams to make the experience lovable and help us deliver solutions that are more than the sum of the parts. As an example flow: Code Insights can lead you to a more detailed Precise Code Intelligence search, from which you might launch a more advanced semantic analysis; from here, you might use Batch Changes to remediate a problem and then document everything you've done in a Search Notebook, monitoring the progress as the fix is rolled out in Code Insights. Another example might be learning about a library in API docs and exploring how it is used over time using Code Insights.
```
Comment (jeanduplessis) [inline content/company/strategy/code-graph/index.md]: > Our product solves real problems today, but investing in the polish, user experience, and cross-feature functionality within our product teams to make the experience lovable and help us deliver solutions that are more than the sum of the parts.

This sentence reads difficult? Or am I just not understanding it properly?

**Review by jeanduplessis — APPROVED** [review]

File: content/company/strategy/code-graph/index.md (hunk 1 above)
Comment (jasonyavorsky) [reply content/company/strategy/code-graph/index.md]: No you're right @jeanduplessis . Better?

```suggestion
Our product solves real problems today, but by investing in polish, user experience, and integrated functionality we can make the experience more lovable and deliver solutions that are "more than the sum of their parts." As an example flow: Code Insights can lead you to a more detailed Precise Code Intelligence search, from which you might launch a more advanced semantic analysis; from here, you might use Batch Changes to remediate a problem and then document everything you've done in a Search Notebook, monitoring the progress as the fix is rolled out in Code Insights. Another example might be learning about a library in API docs and exploring how it is used over time using Code Insights.
```

Comment (jeanduplessis) [reply content/company/strategy/code-graph/index.md]: 👌  much better 🙇

**Pull request merged**